app/src/main/java/com/example/marvelapp/activity/CharacterListActivity.kt

```diff
@@ -54,7 +56,18 @@ class CharacterListActivity : AppCompatActivity(), KoinComponent {
         binding.textViewSubtitleCharactersActivity.text = getString(R.string.character_list_loader)
     }
 
+    private fun showCharacterDetail(characterId: String) {
+        val characterDetailFragment = CharacterDetailFragment.newInstance(characterId)
+        val fragmentManager: FragmentManager = supportFragmentManager
+        characterDetailFragment.show(fragmentManager, TAG)
+    }
+
+    override fun setOnClickListener(characterId: String) {
+        viewModel.onCharacterPressed(characterId)
+    }
+
     companion object {
         fun getInstance(context: Context) = Intent(context, CharacterListActivity::class.java)
+        const val TAG = "CHARACTER_DETAIL_FRAGMENT"
```

Comment:
this TAG should be in the fragmentDialog, not here

Reply:
done